README.MD

```diff
@@ -42,7 +42,7 @@ int main() {
   std::fill(v.begin(), v.end(), 0);
   
   try {
-    Device device;
+    Device device = Devices::findDevice();
```

Comment:
what exactly would happen here if you just call the construktor?

Reply:
Also vom Prinzip her würde es auch funktionieren, wenn nicht der Device-Konstruktor private wäre. Also würde es ein Kompilererror geben

Reply:
hmm glaube einen default construktor faende ich schon toll

Reply:
Ja, kann ich machen, auch wenn ich das so Singelton mäßig eigentlich etwas besser gefunden hätte.
Das Problem ist dabei noch ein bisschen, dass ja auch der Kontext vom Device verwaltet wird, der im Desktrutor dann auch wieder freigegeben werden sollte.
Momentan wird er noch von Devices freigegeben, weil der Device-Desktruktor irgendwie irgendwo aufgerufen wurde. Wenn dann der Destruktor von so einem Device-Singleton aufgerufen wird, ist das natürlich nicht so gut. Dan muss man aber wahrscheinlich nur irgendwas mit Referenzen oder so ändern, dann müsste das funktionieren.

Reply:
hmm also wichtig ist das nicht. lol
finde das nur schön

Reply:
Habe ich gemacht